resources/views/layouts/sidebars/app.blade.php

```diff
@@ -63,9 +65,9 @@
                                 {!! $element['custom_label'] ?: $element['label'] !!}
                             </span>
                         @endif
-                        @if($currentCampaign->boosted())
+                        @if ($currentCampaign->boosted())
                             <ul class="sidebar-submenu">
```

Comment:
I'm confused why the `<ul>` is added here, but not in the others. Adding it on the foreach will add an empty `<ul>` on every loop, which pollutes the HTML, making our CSS wizards sad :)

Reply:
If removed, the sidebar looks like this for all items below the one generated.
![image](https://user-images.githubusercontent.com/72818019/189389461-648d330c-27c2-4ab4-bacd-3cb98f6e4733.png)